examples/mysql-galera/README.md

```diff
@@ -0,0 +1,171 @@
+
+This example was tested on OS X with a Galera cluster running on VMWare using the fine repo developed by Paulo Pires [https://github.com/pires/kubernetes-vagrant-coreos-cluster] and client programs built for OS X.
+
+
+### Running the initial bootstrap node
+
+The basic idea is this: three replication controller, one pod each, and there will only ever be one container per pod since this is not an application where adding more translates to any benefit and is actually undesirable. One of the important design goals of MySQL replication and/or clustering is that you don't want a single-point-of-failure, hence distribute each node or slave across hosts or even geographical locations.
+
+There are only three pods (hence replication controllers) for this cluster. This number could be increased but would require building on a structure that needs to be inevitably made more dynamic, so for this example, 3 is more than sufficient. The number of nodes must always be odd.
+
+Note: Kubernetes best-practices is to pre-create the services for each controller, and this will work fine for this setup, but a faster initial cluster will result if the services are create after the controller for each single-container pod. The reason for this is that if you provide an IP address in the list of ```sst_cluster_address``` that is not yet ready, it'll take that much longer for the SST election process to complete.
+
+First, create the overall cluster service that will be used to connect to the cluster:
+
+
+```kubectl create -f examples/mysql-galera/pxc-cluster-service.yaml```
+
+Create the service and replication controller for the first node:
+
+```kubectl create -f examples/mysql-galera/pxc-node1.yaml```
+
+### Create services and controllers for the remaining nodes
+
+Repeat the same previous steps for ```pxc-node2``` and ```pxc-node3```
+
+When complete, you should be able connect with a mysql client to the IP address
+ service ```pxc-cluster``` to find a working cluster
+
+### Confirming replication controller (or pods) and services are running
+
+Shown below are examples of Using ```kubectl``` from within the ```./examples/mysql-galera``` directory, the status of the lauched replication controllers and services can be confirmed
+
+```
+$ kubectl create -f examples/mysql-galera/pxc-cluster-service.yaml 
+services/pxc-cluster
+
+$ kubectl create -f examples/mysql-galera/pxc-node1.yaml 
+services/pxc-node1
+replicationcontrollers/pxc-node1
+
+$ kubectl create -f examples/mysql-galera/pxc-node2.yaml 
+services/pxc-node2
+replicationcontrollers/pxc-node2
+
+$ kubectl create -f examples/mysql-galera/pxc-node3.yaml 
+services/pxc-node3
+replicationcontrollers/pxc-node3
+
+```
```

Comment:
Cleaned that up as I wanted a an actual output of it being run and explained as such